### PR TITLE
Black

### PR DESCRIPTION
**Pull request by jogo**
Apply black https://black.readthedocs.io/en/stable/

See individual commit messages for more details of each change

### CODE REVIEW

**Review by jparise — COMMENTED** [review]
I think we want to add the the black CI action to the existing `ci.yml` workflow (or move those lint steps out and into a generic `lint.yml` workflow).

File: setup.cfg [hunk 1]
```diff
@@ -44,3 +44,4 @@ markers =
 show-source = True
 max-line-length = 80
```
Comment (jparise) [inline setup.cfg:45]: I think we should also set this to `88`: https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#line-length

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -50,3 +50,11 @@ jobs:
...
         ports:
         - 11211/tcp
+
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: psf/black@stable
+        with:
+          version: "21.5b1"
```
Comment (jparise) [inline .github/workflows/ci.yml:60]: Given that black is already configured in `tox.ini`, you could drop this step entirely and just update the existing lint step above:

```diff
@@ -29,7 +29,7 @@ jobs:
       if: matrix.python-version == '3.9'
       run: |
         pip install tox tox-gh-actions
-        tox -e flake8
+        tox -e black,flake8
     - name: Disable IPv6 localhost
       run: |
         sudo sed -i '/::1/d' /etc/hosts
```

**Review by jparise — COMMENTED** [review]
I think we could also add a `pyproject.toml` configuration that explicitly sets the list of target versions (because pymemcache only supports Python 3.6+):

```toml
[tool.black]
target-version = ['py36', 'py37', 'py38', 'py39']
```

If we wanted  small migration diff, we could also add `skip-string-normalization = true`, but I think it's fine to get that out of the way here.

**Comment by jogo** [comment]
Confirmed black is running correctly https://github.com/pinterest/pymemcache/runs/4411097331?check_suite_focus=true

**Review by jparise — APPROVED** [review]
Nice work!

**Pull request merged**